hugo/assets/scss/mixins/typography.scss

```diff
@@ -59,7 +59,7 @@
     font-size: var(--heading-2-font-size);
 
     @include screen($screen-simple) {
-        --heading-2-font-size: 2rem;
+        --heading-2-font-size: 1.625rem;
```

Comment:
The font size on simple is now smaller than the one on mobile. Is that on purpose?